catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -411,6 +411,17 @@ ResourceResponse getResource(ResourceRequest resourceRequest, boolean isEnterpri
                 LOGGER.info("Unable to download resource", e);
             }
 
+            if (resourceResponse != null) {
+                // must add the request properties into response properties in case the source forgot to
```

Comment:
@lessarderic good points, as a counter argument - the catalogframework should not have to know how the individual sources are implemented and we can't enforce the properties to be maintained within each source.